src/CommunityPatch/Patches/Perks/Cunning/Roguery/SlipIntoShadows.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using HarmonyLib;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.CampaignSystem.SandBox.GameComponents.Map;
+using TaleWorlds.Core;
+using static CommunityPatch.HarmonyHelpers;
+
+namespace CommunityPatch.Patches.Perks.Cunning.Roguery {
+
+  public sealed class SlipIntoShadows : PatchBase<SlipIntoShadows> {
```

Comment:
Add `Patch` as suffix for both class name and file, to follow the same pattern as the others patches in the mod

Reply:
Sorry! I'll have that fixed